src/command.ts

```diff
@@ -535,8 +535,31 @@ export function parseAndSpawnCommand(state: CommandBuilderState) {
     throw new Error("A command must be set before it can be spawned.");
   }
 
+  const toLog = [];
   if (state.printCommand) {
-    state.printCommandLogger.getValue()(colors.white(">"), colors.blue(state.command));
+    toLog.push(
+      colors.white(">"),
+      colors.blue(state.command),
+    );
+    if (state.cwd) {
+      toLog.push(
+        colors.white("\ncwd:"),
+        colors.cyan(state.cwd),
+      );
+    }
+    if (Object.keys(state.env).length != 0) {
+      toLog.push(colors.white("\nenv:"));
+      for (const [key, value] of Object.entries(state.env)) {
+        if (value) {
+          toLog.push(
+            colors.cyan(key),
+            colors.white("="),
+            colors.cyan(value),
+          );
+        }
+      }
+    }
+    state.printCommandLogger.getValue()(...toLog);
```

Comment:
Sorry, this is way too much information and beyond the scope of what I'd like to support here. For debugging purposes you can print the cwd by doing: `pwd` or if you need the environment variables you can do `deno eval 'console.log(Deno.env.toObject())''. I would accept a PR to add https://man7.org/linux/man-pages/man1/printenv.1.html